src/Cake.Common/Tools/ReportGenerator/ReportGeneratorSettings.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Cake.Core.Annotations;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.ReportGenerator
+{
+    /// <summary>
+    /// Contains settings used by <see cref="ReportGeneratorRunner"/>.
+    /// </summary>
+    [CakeAliasCategory("ReportGenerator")]
+    public sealed class ReportGeneratorSettings
```

Comment:
>@patriksvensson said...
All tools should have an optional tool path. Not everyone has one, but they should :)¨

Should there be a base Settings Class that contains the ToolPath property, and each Tool Settings class derives from this?